pkg/fakerp/config.go

```diff
@@ -76,7 +76,7 @@ func NewConfig(log *logrus.Entry, needRegion bool) (*Config, error) {
 	return &c, nil
 }
 
-func GetPluginConfig() (*api.PluginConfig, error) {
+func GetPluginConfig(isAdmin bool) (*api.PluginConfig, error) {
 	tc := api.TestConfig{
 		RunningUnderTest:      os.Getenv("RUNNING_UNDER_TEST") != "",
```

Comment:
do you need to rebase? this line is `os.Getenv("RUNNING_UNDER_TEST") == "true"` in master

Reply:
Not necessary, my branch is not on top of HEAD but there is no conflict.